src/tvheadend/ChannelTuningPredictor.cpp

```diff
@@ -74,7 +74,7 @@ uint32_t ChannelTuningPredictor::PredictNextChannelId(uint32_t tuningFrom, uint3
    * it will point at the channel we should tune to */
   std::set<ChannelPair>::iterator predictedIt = m_channels.cend();
 
-  if (std::next(fromIt, 1) == toIt || toIt->second == firstNum) {
+  if (fromIt == m_channels.cend() || std::next(fromIt, 1) == toIt || toIt->second == firstNum) {
```

Comment:
@Jalle19 On my setup on the initial/first tune, the tuneFrom channelid parameter of `PredictNextChannelId` is zero which causes  `GetIterator` to return `end()` which in turn crashes if `std::next` is used on it. 
What should `tuningFrom` be set to initially ?